site/src/pages/how-to-set-up-google-analytics.mdx

```diff
@@ -43,7 +43,7 @@ You will get a snippet from Google called the "Global Site Tag (gtag.js)", which
 
 <img src="/images/tutorials/install-google-analytics/step-4.png" alt="Navigate to Admin section on Google Analytics" style="width: 100%"/>
 
-```
+```html
```

Comment:
Since this doesn't have syntax highlighting yet, feel free to remove these and I'll approve your PR :)

Reply:
For me this syntax highlighting works 😳
See:

<img width="1072" alt="Bildschirmfoto 2020-04-29 um 20 49 47" src="https://user-images.githubusercontent.com/26602940/80634658-15c35000-8a5b-11ea-8774-ec8755a81eae.png">

Reply:
@0xflotus You're looking at this in GitHub. But if you open the actual help center, you'll see there is no syntax highlighting: https://deploy-preview-2--gatbsy-theme-help-center.netlify.app/articles/how-to-set-up-google-analytics/

I'm going to remove these from this PR before merging the changes, and then it can be reintroduced in the other PR that's open #3 which adds syntax highlighting.